### DISCUSSION TITLE
Cannot find module '@metaplex-foundation/js'

### DISCUSSION THREAD

**Discussion by M-Rb3** [metaplex-foundation/js#179] [Q&A] [answered]
Hi, I hope you all doing great, I am trying to import the model in react app, but I am getting  this error
`Module not found: Can't resolve '@metaplex-foundation/js'`
![image](https://user-images.githubusercontent.com/68287884/180179179-ee8d4fa2-88f3-4d00-af7a-252a9e50b1ee.png)
![image](https://user-images.githubusercontent.com/68287884/180179418-223ef98d-e4f9-4f24-b69f-bee230900b98.png)

what am I doing wrong?
Thanks in advance

**Comment by lorisleiva** [comment, marked as answer]
Sorry I've messed up version 0.12.4 the dist folder is missing. Working on a new version as we speak. 🤦‍♂️

**Reply by lorisleiva** [reply]
Okay, we should be back to normal from `0.12.5`! Sorry again about that.

**Reply by M-Rb3** [reply]
okay, thanks @lorisleiva 
I tried downgrading the version to 0.12.3, but I am getting this new error

./node_modules/@metaplex-foundation/js/dist/esm/plugins/tokenModule/createToken.mjs
Can't import the named export 'ACCOUNT_SIZE' from non EcmaScript module (only default export is available)

**Reply by M-Rb3** [reply]
after following the example here, it solved the issue 
https://github.com/metaplex-foundation/js-examples/tree/main/getting-started-react-cra4
